osgi.specs/docbook/core/003/framework.module.xml

```diff
@@ -2231,10 +2231,58 @@ package-names   ::= package-name                // See <xref
       and implementation in the same package making it impossible to
       substitute the API packages.</para>
 
-      <para>Importing an exported package must use a version range according
+      <para>Importing an exported package <emphasis role="bold">must</emphasis> use a version range according
       to its compatibility requirements, being either a consumer or a provider
       of that API. See <xref linkend="framework.module-semantic.versioning"/>
       for more information.</para>
+
+      <para>Also see the section about <xref linkend="framework.module-import.export.same.package"/> 
+      for more information about how the resolver handles this and what problems could arise with substitution.</para>
+
+      <note>
+      <para>
+		  As a recommendation: <emphasis role="bold">Library bundles</emphasis> 
```

Comment:
I find this a bit superfluous, especially as there is no definition of "library bundle" anywhere in the spec. Also it implies it should be used for "other bundles" I simply would not suggest using it at all, for me it would be enough to state that it is possible (e.g. not disallowed!) to import/export the same package.

Reply:
> I think the previous paragraphs are better rewritten instead of adding to verbose explanation afterwards.

I have pushed a rewrite of the previous paragraph and incorporated some parts of my notes in it. This reduced the size of my additional note.
I rephrased the existing section to now emphasize that substitution is not recommended and only useful in rare cases. Let's see how that works.

> I find this a bit superfluous, especially as there is no definition of "library bundle" anywhere in the spec. 

I rewrote this part. In the previous version of the spec the word "libraries" was (and is still ) used though. I rephrased it to "general/purpose java libraries.